packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -218,7 +218,11 @@ class Auth {
                 siteId: config.siteId
             },
             throwOnBadResponse: true,
-            fetchOptions: config.fetchOptions
+            // We need to set credentials to 'same-origin' to allow cookies to be set. This is required as SLAS calls return a dwsid cookie for hybrid sites.
+            fetchOptions: {
+                credentials: 'same-origin',
+                ...config.fetchOptions
```

Comment:
if we don't want `credentials` to be overridden by user input, maybe line 224 should be moved above line 223
```suggestion
                ...config.fetchOptions,
                credentials: 'same-origin'
```

Reply:
I think we will let the user override if needed as this is only required for hybrid sites

Reply:
Yeah allowing an option to override makes more sense since customers may have custom implementations around cookies for non-hybrid sites which might break something.
So having the ability to disable cookie passthrough without another sdk release is good.

Reply:
@unandyala Do we want to/need to document why we're introducing this ?

Reply:
Other than change log may be we can add to the hybrid document to ensure this not removed/modified